tracerite/trace.py

```diff
@@ -146,7 +146,7 @@ def extract_frames(tb, suppress_inner=False) -> list:
                 cls = next(
                     v.__class__ if n == 'self' else v
                     for n, v in frame.f_locals.items()
-                    if n in ('self', 'cls')
+                    if n in ('self', 'cls') and v is not None
                 )
                 function = f'{cls.__name__}.{function}'
```

Comment:
What does this change address? A comment in code might be useful.

Reply:
One of the uvloop tabs showed: `NoneType.run` instead of just `run`

Reply:
It would appear to be a case with non-Python code then.